core/src/integrationTest/kotlin/ch/ergon/dope/integrationTest/TestCouchbaseDatabase.kt

```diff
@@ -0,0 +1,108 @@
+package ch.ergon.dope.integrationTest
+
+import ch.ergon.dope.integrationTest.TestCouchbaseDatabase.testBucket
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.fromable.Bucket
+import ch.ergon.dope.resolvable.fromable.UnaliasedBucket
+import ch.ergon.dope.validtype.BooleanType
+import ch.ergon.dope.validtype.NumberType
+import ch.ergon.dope.validtype.StringType
+import com.couchbase.client.kotlin.Cluster
+import com.couchbase.client.kotlin.query.QueryResult
+import com.couchbase.client.kotlin.query.execute
+import kotlinx.coroutines.runBlocking
+import org.testcontainers.couchbase.BucketDefinition
+import org.testcontainers.couchbase.CouchbaseContainer
+import org.testcontainers.utility.DockerImageName
+import kotlin.test.assertEquals
+import kotlin.time.Duration.Companion.seconds
+
+const val BUCKET = "testBucket"
+
+object TestCouchbaseDatabase {
+    private val container = CouchbaseContainer(
+        DockerImageName.parse("couchbase/server:community-7.6.2"),
```

Comment:
Do we have to hardcode the version or could we just use the `latest`?

Reply:
I could, but I prefer using a specific version for reproducibility. I can make it a constant, though.